openlibrary/macros/Follow.html

```diff
@@ -10,7 +15,7 @@
       <input type="hidden" value="$publisher" name="publisher"/>
       <input type="hidden" value="$custom_request_path" name="redir_url"/>
       $ css_treatment = 'delete' if following else 'primary'
-      <button type="submit" class="cta-btn cta-btn--$(css_treatment)">$_("Unfollow" if following else "Follow")</button>
+      <button type="submit" class="cta-btn cta-btn--$(css_treatment)" data-i18n="$json_encode(i18n_strings)">$_("Unfollow" if following else "Follow")</button>
```

Comment:
```suggestion
      <button type="submit" class="cta-btn cta-btn--$(css_treatment)" data-i18n="$json_encode(i18n_strings)">$(_("Unfollow") if following else _("Follow"))</button>
```
Somehow, this is creating a "UnfollowFollow" entry in `messages.pot`, [here](https://github.com/internetarchive/openlibrary/blob/009b67e7946f3e156891d84dc4c37a34173ddf46/openlibrary/i18n/messages.pot#L303-L305).

This should probably be fixed in a separate PR.  Will open an issue after this is merged.